libraries/preferences/api/src/main/kotlin/io/element/android/features/preferences/api/store/PreferencesStore.kt

```diff
@@ -28,6 +28,9 @@ interface PreferencesStore {
     suspend fun setCustomElementCallBaseUrl(string: String?)
     fun getCustomElementCallBaseUrlFlow(): Flow<String?>
 
+    suspend fun setPrivateReadReceiptsEnabled(enabled: Boolean)
+    fun isPrivateReadReceiptsEnabled(): Flow<Boolean>
```

Comment:
It's a bit annoying me that this settings is stored in `PreferencesStore`, which should be used for global settings only. The setting `isPrivateReadReceiptsEnabled` should have a Session scope.

That said, IIRC, we do not have (yet) a store for session preferences, *and* we do not support multi sessions yet. Hence, I think logging out and logging in again will keep the setting, and it's probably an unexpected side effect from the user POV.

Reply:
I'll see how difficult it is to create a session-scoped preference store.